packages/vats/src/vat-board.js

```diff
@@ -1,7 +1,23 @@
-import { Far } from '@endo/far';
-import { makeBoard } from './lib-board.js';
+// @ts-check
+import { M, prepareExo } from '@agoric/vat-data';
+import { prepareBoardKit } from './lib-board.js';
 
-export function buildRootObject() {
-  const board = makeBoard();
-  return Far('board', { getBoard: () => board });
+const RootI = M.interface('VatBoard', {
+  getBoard: M.call().returns(M.remotable()),
+});
+
+/**
+ * @param {unknown} _vatPowers
+ * @param {unknown} _vatParameters
+ * @param {import('@agoric/vat-data').Baggage} baggage
+ */
+export function buildRootObject(_vatPowers, _vatParameters, baggage) {
+  const makeBoardKit = prepareBoardKit(baggage);
```

Comment:
I think anything accepting a baggage argument to write into should also accept a key rather than hard-coding it.
```suggestion
  const makeBoardKit = prepareBoardKit(baggage, 'Board');
```

Reply:
Disagree, but I see an extensive internal slack discussion, so will continue there. Whatever the outcome, someone should publicly summarize.